<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="useable_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="8,92">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="navi_widget" native="true">
        <property name="styleSheet">
         <string notr="true">#navi_widget {
background-color: white;
border-bottom: 1px solid #CBD1E0;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,11,83,6,1">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>12</number>
           </property>
           <item>
            <widget class="QLabel" name="icon_label">
             <property name="styleSheet">
              <string notr="true">#icon_label {
width: 26px;
height: 26px;
min-width: 26px;
min-height: 26px;
max-width: 42px;
max-height: 42px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="name_label">
             <property name="styleSheet">
              <string notr="true">#name_label {
font:  15px Microsoft YaHei ; 
color: #333333;
}</string>
             </property>
             <property name="text">
              <string>BIGJPG</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>877</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true">#min_btn, #close_btn {
width:14px;			
height:14px;	
min-width:14px;    
min-height:14px;                   
max-width:24px;
max-height:24px;
}</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <item row="0" column="1">
             <widget class="QToolButton" name="min_btn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QToolButton" name="close_btn">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="central_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="11,89">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_3 {
border: none;
background-color: #4586E4;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,2,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QListWidget" name="option_list_widget">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>56</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">#option_list_widget {
border: none;
background-color: #4586E4;
font-size: 14px;
font-weight: bold;
}
</string>
              </property>
              <property name="autoScrollMargin">
               <number>16</number>
              </property>
              <property name="movement">
               <enum>QListView::Static</enum>
              </property>
              <property name="flow">
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="resizeMode">
               <enum>QListView::Fixed</enum>
              </property>
              <property name="layoutMode">
               <enum>QListView::SinglePass</enum>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="viewMode">
               <enum>QListView::ListMode</enum>
              </property>
              <property name="itemAlignment">
               <set>Qt::AlignRight</set>
              </property>
              <property name="currentRow">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="function_stack_widget">
           <property name="styleSheet">
            <string notr="true">#function_stack_widget{
background-color: white;
}</string>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="midLineWidth">
            <number>0</number>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
